deployment/misc/sdklocalmethodrunner.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
```

Comment:
Is this basically a local method runner ported to Python?

Reply:
It's a script to call LocalMethodRunner on multiple environments outside of KBase as well as in KBase. I don't see this being merged anytime soon